lms/djangoapps/courseware/date_summary.py

```diff
@@ -326,7 +326,8 @@ class CourseEndDate(DateSummary):
     def description(self):
         if self.current_time <= self.date:
             mode, is_active = CourseEnrollment.enrollment_mode_for_user(self.user, self.course_id)
-            if is_active and CourseMode.is_eligible_for_certificate(mode):
+            has_verified_mode = CourseMode.mode_for_course(self.course_id, CourseMode.VERIFIED)
```

Comment:
Thinking about it, this line might be too restrictive. I'm not sure if other enrollment tracks offer certificates (like Professional or even Masters only courses that offer certificates)

Reply:
Yeah. Might be valuable to ask Deen about it, since during standup he seemed to have a more well-constructed picture in his mind (as compared to me, anyway!) of what the menagerie of different Master's course setups look like w.r.t. certificates. (Apologies for giving you PR feedback that sends you back into JIRA, not very helpful!)